src/secp256k1/configure.ac

```diff
@@ -20,7 +20,7 @@ AC_PATH_TOOL(STRIP, strip)
 AX_PROG_CC_FOR_BUILD
 
 if test "x$CFLAGS" = "x"; then
-  CFLAGS="-O3 -g"
+  CFLAGS="-g"
```

Comment:
This just changes the default. Having optimization enabled in a dev build is not the best idea.